Jenkinsfile

```diff
@@ -31,6 +31,16 @@ def mavenEnv(Map params = [:], Closure body) {
   }
 }
 
+@NonCPS
+def parsePlugins(plugins) {
+  def pluginsByRepository = [:]
+  plugins.each { plugin ->
+    def splits = plugin.split('\t')
+    pluginsByRepository[splits[0]] = splits[1].split(',')
+  }
+  pluginsByRepository
+}
+
```

Comment:
The new format of the file is `group/repository\tplugin1,plugin2,plugin3`. This parsing function is marked `@NonCPS` for performance reasons, as serializing each line to disk for a 200-line list would be inefficient.